src/pages/home/components/titleItem/titleItem/ItmeImageStroke.jsx

```diff
@@ -103,7 +71,7 @@ const ItmeImageStroke = ({ itemInfo }) => {
           />
         </foreignObject>
         {(divideNum === 3 ? threeStrockInfo : oneStrockInfo).map((element) => {
-          if (element.key <= usageCount)
+          if (element.key <= divideNum - itemInfo.usageCountPerDay)
```

Comment:
```suggestion
          if (element.key <= divideNum - usageCount)
```

여기를 먼저 고치고 고민하다 위에를 나중에 수정한거라 이 부분은 불필요한 수정이긴 한데 같이 커밋되었네용..